apps/edxapp/templates/services/nginx/bc_collectstatic.yml.j2

```diff
@@ -0,0 +1,59 @@
+{%- from "apps/edxapp/templates/services/cms/macros/image.yml.j2" import image_name -%}
+{%- set flag="collectstatic" -%}
```

Comment:
I don't see the interest to have this definition here.

Reply:
Avoid repetition

Reply:
repeat what ? You repeat `flag` instead of `"collectstatic"` ? This is useless IMO

Reply:
> ok